PeerioTechnologies/peerio-mobile
# fix: channel info actions fixed (make admin, remove participant, remove invite)

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/7429/mobile-medcryptor-room-allows-multiple-remove-actions
https://app.clubhouse.io/peerio/story/7427/mobile-medcryptor-room-can-t-remove-mc-member

#### Testing instructions  
All channel info actions should work for Peerio + MC: make/demote admin, remove participant, remove invite. 

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
